app/pb_kits/playbook/packs/site_styles/_site-style.scss

```
@@ -25,6 +25,22 @@ body {
     &.dark{
     background: tint(#0a0527, 10);
     }
+    .mobile-popover-nav {
+      display: none;
+    }
+    @include break_at(breakpoint("lg")) {
+      .full-size-nav {
+        display: none;
+      }
+      padding: 20px 30px;
```

Can you use our tokens?

```suggestion
      padding: $space_md $space_lg;
```

---

Nav spacing is really precise doesn't work well with our spacing variables.  I'll use a differential here though (`$space_lg - 2`).